src/Components/Todo/index.jsx

```diff
@@ -37,24 +37,26 @@ const Todo = () => {
   const { handleChange, handleSubmit } = useForm(addItem, defaultValues);
 
   async function addItem(item) {
-
-    // item.id = uuid();
+try {
     item.complete = false;
     const config = {
       baseURL: 'https://api-js401.herokuapp.com/api/v1/todo',
       method: 'post',
       data: item
     };
-    let response = axios(config);
+    let response = await axios(config);
     console.log('item', response.data);
     setList([...list, item]);
```

Comment:
Also, make sure you are adding the item object you get back from the server response, because we need the `item._id` that the server generates. Right now we can't update or delete until we refresh the page.
```js
setList([...list, response.data]);